.github/workflows/perf_sysbench.yml

```diff
@@ -23,15 +23,28 @@ jobs:
       fail-fast: false
 
     steps:
-      - name: set PATH to GIT of the newer version 2.9.0
-        run: echo "/usr/local/git/bin" | tee -a $GITHUB_PATH
-      - name: cleanup workspace
-        run: docker run -w /source -v ${PWD}:/source -i centos:8 /bin/bash -c "chown -R $(id -u):$(id -g) * .[^.]*"
-      - uses: actions/checkout@v1
-      - name: test
-        env:
-          BENCH: 'sysbench'
-        uses: ./.github/actions/perf
+#      TODO: migrate to v2 when Git 2.18 or higher would be available
+      - name: checkout tarantool
+        uses: actions/checkout@v1
+        with:
+          submodules: true
+      - name: checkout sysbench repo
+        uses: actions/checkout@v2.3.4
+        with:
+          path: sysbench
+          repository: tarantool/sysbench
+      - name: tarantool build
+        run: |
+          cmake . -DENABLE_DIST=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo && make -j 4
+      - name: configure perf benchmark
+        run: |
+          ./autogen.sh
+          ./configure --with-tarantool --without-mysql
+          make -j 4
+        working-directory: ${{ github.workspace }}/sysbench
```

Comment:
Not sure that we need to specify `${{ github.workspace }}`, relative paths should work fine.

Reply:
changed to relative paths